### PR TITLE
Updates Crossover Compatability list + adds r2modman page

### PR DESCRIPTION
**Pull request by hahayupgit**
Adds Epic Games Launcher, Rockstar Games, and Fortnite to crossover compatibility list. Also adds a page for r2modman.

### CODE REVIEW

File: src/cx.md [hunk 1]
```diff
@@ -2,21 +2,24 @@
...
+| EA App Support                  | ❌      | ✅         |
+| Epic Games Launcher Support     | ❌      | ✅         |
+| Battle.net Support              | ❌      | ✅         |
+| Rockstar Games Launcher Support | ❌      | ✅         |
+| Ubisoft Connect Support         | ❌      | ✅         |
+| OOTB GStreamer Support          | ❌      | ✅         |
+| Wine 9 Support                  | ❌      | ✅         |
+| Proper Technical Support        | ❌      | ✅         |
+| Future Updates to Wine          | ❌      | ✅         |
+| Fortnite Support                | ❌      | ❌         |
```
Comment (IsaacMarovitz) [inline src/cx.md:23]: Funny, but let's not make this table longer than it already is

File: src/game-support/r2modman.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+## r2modman
+
+{{#template ../templates/rating.md status=Silver date=2/27/24 installs=Yes opens=Yes}}
```
Comment (IsaacMarovitz) [inline src/game-support/r2modman.md:3]: Date should be `02/27/24`

File: src/game-support/r2modman.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (IsaacMarovitz) [inline src/game-support/r2modman.md:4]: Missing `## Setup` header

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

**Comment by hahayupgit** [comment]
Should be updated with all fixes complete

**Review by IsaacMarovitz — APPROVED** [review]

**Pull request merged**